src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -132,15 +132,15 @@ function AdditionalDetailsStep({walletAdditionalDetails, translate, currentUserP
      */
     const activateWallet = (values) => {
         const personalDetails = {
-            phoneNumber: parsePhoneNumber(values.phoneNumber, {regionCode: CONST.COUNTRY.US}).number.significant,
-            legalFirstName: values.legalFirstName,
-            legalLastName: values.legalLastName,
-            addressStreet: values.addressStreet,
-            addressCity: values.addressCity,
-            addressState: values.addressState,
-            addressZip: values.addressZipCode,
-            dob: values.dob,
-            ssn: values.ssn,
+            phoneNumber: parsePhoneNumber(values.phoneNumber, {regionCode: CONST.COUNTRY.US}).number.significant || '',
+            legalFirstName: values.legalFirstName || '',
+            legalLastName: values.legalLastName || '',
+            addressStreet: values.addressStreet || '',
+            addressCity: values.addressCity || '',
+            addressState: values.addressState || '',
+            addressZip: values.addressZipCode || '',
+            dob: values.dob || '',
+            ssn: values.ssn || '',
```

Comment:
NAB: I think we prefer `?? ''` in these cases

Reply:
This file is in JS and turns out `??` is still forbidden in JS 😅 
![image](https://github.com/Expensify/App/assets/46095609/ace78f77-7f71-4950-a931-cb1752b28945)